test/test_artifacts/v0/sagemaker-headless-execution-driver.test.Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+
+# Execute the unit tests for sagemaker-headless-execution-driver
```

Comment:
Nit: statement below is just an import; not executing any tests. Are there unit tests in the package we can run from here?

Reply:
Yes there are unit test. I will fix it